ts/src/ellipx.ts

```diff
@@ -0,0 +1,1583 @@
+        //                 "tz": "UTC",
+        //                 "full": "1728712510669096",
+        //                 "unixms": "1728712510669"
+        //             }
+        //         }
+        //     ],
+        //     "paging": {
+        //         "page_no": 1,
+        //         "count": "1",
+        //         "page_max": 1,
+        //         "results_per_page": 20
+        //     }
+        // }
+        const data = this.safeValue (response, 'data', []);
+        return this.parseOrders (data, market, since, limit);
+    }
+
+    /**
+     * @method
+     * @name ellipx#fetchOrders
+     * @description fetches information on multiple orders made by the user
+     * @param {string} symbol unified market symbol of the market orders were made in
+     * @param {int|undefined} since timestamp in ms of the earliest order
+     * @param {int|undefined} limit the maximum amount of orders to fetch
+     * @param {object} params extra parameters specific to the exchange API endpoint
+     * @returns {Order[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async fetchOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        return await this.fetchOrdersByStatus (undefined, symbol, since, limit, params);
+    }
+
+    /**
+     * @method
+     * @name ellipx#fetchOpenOrders
+     * @description fetches information on open orders made by the user
+     * @param {string} symbol unified market symbol of the market orders were made in
+     * @param {int|undefined} since timestamp in ms of the earliest order
+     * @param {int|undefined} limit the maximum amount of orders to fetch
+     * @param {object} params extra parameters specific to the exchange API endpoint
+     * @returns {Order[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async fetchOpenOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        return await this.fetchOrdersByStatus ('open', symbol, since, limit, params);
+    }
+
+    parseOrder (order, market = undefined): Order {
+        const id = this.safeString (order, 'Market_Order__');
+        const timestamp = this.safeInteger (this.safeValue (order, 'Created'), 'unixms');
+        const typeResponse = this.safeString (order, 'Type');
+        let side = 'sell';
+        if (typeResponse === 'bid') {
+            side = 'buy';
+        }
+        const status = this.parseOrderStatus (this.safeString (order, 'Status'));
+        const amount = this.parseAmount (order['Amount']);
+        const price = this.parseAmount (order['Price']);
+        const type = (price === undefined) ? 'market' : 'limit';
+        const executed = this.parseAmount (order['Executed']);
+        const filled = executed;
+        const remaining = (amount !== undefined && filled !== undefined) ? amount - filled : undefined;
```

Comment:
we shouldn't do raw calculations like this, we should use Precise.X` to do them, please check other implementations as a reference.

Reply:
as this from `{v: e: f:}` response so it need to rework on some function.

Reply:
fix with revise the `parseAmount` method using `Precise`.